src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -668,34 +693,16 @@ CharMap =
 # There are a lot of good resources for finding MIME types on the Internet, one
 # of which is http://www.iana.org/assignments/media-types/media-types.xhtml
 #
-# Each of the "Supported" lines contains the following parameters separated by
-# spaces or tabs:
-#
-#    f:         Regular expression to match the format of a file, see the list
-#               above. This parameter is mandatory.
-#    v:         Regular expression to match the video codec, see the list above.
-#    a:         Regular expression to match the audio codec, see the list above.
-#    m:         The MIME type to send to the renderer. Use of this parameter is
-#               strongly recommended. If it is not set, an estimated value will
-#               be assigned.
-#    n:         The maximum number of audio channels.
-#    s:         The maximum audio sample frequency (Hertz).
-#    b:         The maximum bitrate (bits per second) of the whole file.
-#    w:         The maximum video/image width (pixels).
-#    h:         The maximum video/image height (pixels).
-#    qpel:      Quarter pixel codec setting (yes or no).
-#    gmc:       The number of global motion compensation warp points.
-#    gop:static Whether the only supported MPEG-2 GOP type is static.
+# SUPPORTED SUBTITLES TAGS
 #
-
-# Supported video formats:
-# Supported audio formats:
-
 # A comma-separated list of supported text-based external subtitles formats
 # for streaming together with streaming (not transcoded) video.
+# Ignored if MediaInfo = true.
```

Comment:
If these old settings are ignored if `MediaInfo = true` then does that mean we need to go update all the existing renderer configs that use these lines and mediainfo=true so we don't break their subtitles support?

Reply:
@SubJunk that is why I asked for testing and review. It should override the existing subtitles support. I left the current setting untouched to be here for using FFmpeg instead of MediaInfo.

Reply:
@valib ok I will test what happens when `SupportedExternalSubtitlesFormats` and `MediaInfo = true` are defined.